bignum.c

```diff
@@ -237,6 +237,10 @@ sexp sexp_bignum_fxrem (sexp ctx, sexp a, sexp_sint_t b) {
   b0 = (b >= 0) ? b : -b;
   for (i=len-1; i>=0; i--) {
     n = (n << sizeof(sexp_uint_t)*8) + data[i];
+    // ISSUE_296 - Division by zero, assigned line 237 if b >= 0
+    if (b0 == 0) {
+      return sexp_read_error(ctx, "division by zero", SEXP_NULL, NULL);
```

Comment:
This should be checked outside the loop, and return a user exception, not read error.  Actually, it should be checked in sexp_remainder along with the divide by SEXP_ONE case.